flang/runtime/exceptions.cpp

```diff
@@ -12,9 +12,26 @@
 #include "terminator.h"
 #include <cfenv>
 
+// Some system don't define those exceptions, although they are mandated by
+// c++11 standard (e.g. musl, emscripten).
+#ifndef FE_INVALID
+#define FE_INVALID 0
+#endif
```

Comment:
I don't think those are conforming definitions.

Reply:
I moved to crashing if we want to report a failure that's not supported by the runtime because that shouldn't happen. Thanks for the insight.